TestCases.md

```diff
@@ -0,0 +1,487 @@
+3. Check that the page displays a 24-word mnemonic phrase in a numbered grid. All random.
+
+**Back button**
+
+1. Verify the presence and correct functionality of a "Back" button.
+2. Verify that clicking "Back" navigates the user back to the main page.
+3. Ensure that no mnemonic phrase information is retained when returning to the main page.
+
+**Generate New Mnemonics button**
+
+1. Verify the presence of a "Generate New Mnemonics" button.
+2. Verify that clicking "Generate New Mnemonics" generates a new set of 24 words.
+3. Ensure that the new mnemonic phrase differs from the previously generated one.
+
+**Copy to Clipboard button**
+
+1. Verify the presence of a "Copy to Clipboard" button.
+2. Verify that clicking "Copy to Clipboard" copies the 24-word mnemonic phrase to the clipboard.
+3. Ensure that a success message is shown after copying.
+4. Test that the copied mnemonic phrase can be pasted correctly into another application.
+5. Test that the copied mnemonic phrase can be pasted correctly into the wallet app and successfully open the correct wallet.
+
+**Next step button**
+
+1. Verify the presence and correct functionality of a "Next Step" button.
+2. Verify that clicking "Next Step" navigates the user to the next appropriate page: Mnemonics confirmation step
+3. Verify that the 24-word mnemonic phrase is carried over correctly to the next step.
+
+## Mnemonic Confirmation Page
+
+1. Verify the mnemonic confirmation page loads successfully.
+2. Check that the page displays a 24-word mnemonic phrase from the previous step.
+3. Check that the mnemonic phrase consists of 24 numbered slots, with 20 words correctly filled and 4 slots empty.
+4. Ensure the 4 missing words are listed above the grid for drag-and-drop.
+5. Verify that the 4 missing words can be dragged and dropped into the empty slots on the grid.
+6. Ensure that each word can only be placed in one slot.
+7. Ensure that only one word can be placed in one slot at a time. If there was already a word, it should pop out to the initial top position above the grid.
+8. Check that a word, once placed, remains in the slot unless dragged out again.
+9. Ensure that visually all words are inserted in the same way (with no additional space or visual changes)
+10. Ensure the user can easily drag the word out of the incorrect slot and try again.
+11. Verify that correctly placing the words into their slots provides visual feedback.
+12. Ensure that an error message or visual indication is provided if the words are placed in the wrong slots, and the "Next Step" button remains inactive.
+
+**Back button**
+
+1. Verify the presence of a "Back" button.
+2. Verify that clicking "Back" navigates the user back to the mnemonic phrase page.
+3. Ensure that the mnemonic phrase is retained when returning to the mnemonic phrase page.
+4. Ensure that clicking 'Back' clears all filled words and prompts the user to fill in the 4 words again.
```

Comment:
These are 4 different words => it pushes a user to save the mnemonic somewhere (a user cannot just click back&next to fill the missing words easily)

Reply:
applied, thank you 🙏